### ISSUE TITLE
Various moves missing in later generations.

### ISSUE THREAD

**Issue by ImmortalZypther** [nerd-of-now/NCP-VGC-Damage-Calculator#13]
After looking through move_data.js, it looks like a bunch of moves are removed because they change across versions but are never added back?
![image](https://user-images.githubusercontent.com/21045587/214941378-1fc88188-1309-4f67-884e-8b1b4a62b948.png)

The deletions also sometimes don't exactly line up with when the actual change happens. You can find "delete MOVES_ADV['BubbleBeam'];" but Bubble Beam only changed from gen 2 onwards, not starting with gen 3 (it's only a change to its chance for lowering speed, which doesn't affect damage calculations).

**Comment by nerd-of-now** [comment]
A lot of this data is mostly how it was before I took over (based on where it was deleted, it was probably from before the damage calculator branched off from Showdown). I haven't really added them back because they haven't had meta relevance in VGC ever (except Razor Leaf which was probably why it was added back in BW). I can get to adding them back once I get through adding the rest of the altogether missing moves.
